frame/support/procedural/src/pallet/parse/call.rs

```diff
@@ -55,14 +56,17 @@ pub struct CallVariantDef {
 	pub args: Vec<(bool, syn::Ident, Box<syn::Type>)>,
 	/// Weight formula.
 	pub weight: syn::Expr,
+	/// PoV size formula, if supplied.
+	pub pov_size: Option<syn::Expr>,
 	/// Docs, used for metadata.
 	pub docs: Vec<syn::Lit>,
 }
 
 /// Attributes for functions in call impl block.
-/// Parse for `#[pallet::weight(expr)]`
-pub struct FunctionAttr {
-	weight: syn::Expr,
+/// Parse for `#[pallet::weight(expr)]` or `#[pallet::pov_size(expr)]`
```

Comment:
currently `pallet::weight` give all the dispatch info except pov_size. I know its name is weird, but maybe we should merge everything into one attribute ?
but it is probably better as you do now.